listenbrainz/webserver/views/user.py

```diff
@@ -180,6 +181,31 @@ def artists(user_name):
     )
 
 
+@user_bp.route('/<user_name>/delete-listen', methods=['POST'])
+@login_required
+def delete_listen():
+    """ Delete a particular listen from the currently logged-in user's listen history.
+
+    This checks for the correct authorization token and deletes the listen.
+    """
+    if request.form.get('token') and (request.form.get('token') == current_user.auth_token):
+        try:
+            recording_msid = request.form.get('recording_msid')
+            if recording_msid is None:
+                log_raise_400("%s Recording MSID missing.")
```

Comment:
You're missing the `% recording_msid` here.

Reply:
Actually the %s is redundant. I have removed it.